SPF check whitelisted domains

paukstis opened the pull request:
Hi Gilles, I think it would make sense to do SPF check whitelisted domains, otherwise they might be abused by spammers, i.e. without this check as soon as "@gmail.com" gets whitelisted anyone with mail-from "@gmail.com" will bypass greylisting.

paukstis committed:
Always SPF check whitelisted domains

It probably makes sense to do SPF check with whitelisted domains, otherwise they might be abused by spammers, i.e. as soon as "@gmail.com" gets whitelisted anyone with mail-from "@gmail.com" will bypass greylisting.

paukstis committed:
Merge pull request #1 from paukstis/paukstis-patch-1

Always SPF check whitelisted domains

poolpOrg commented:
makes sense, I'm unable to test today though but will do as soon as I can and merge, thanks !

paukstis committed:
add SPF valid domain autowhitelist mode

spfdomainwl switch to enable SPF valid domain autowhitelist mode, this puts domain with valid SPF automatically into whitelist.

paukstis committed:
fixing pointers, thanks Go, C would go blind

paukstis commented:
Sorry, didn't realize I was on master branch with this PR when adding another thing, the idea would be to start immediately whitelisting SPF valid domains, without going through greylist, what do think ? Move it to separate PR ?

paukstis commented:
undo, make separate PRs
